src/commtrack.js

```diff
@@ -89,21 +100,11 @@ define([
                 }),
                 logicSection
             ],
-            Transfer: [
-                _.extend({}, basicSection, {
-                    properties: [
-                        "nodeID",
-                        "src",
-                        "dest",
-                        "sectionId",
-                        "entryId",
-                        "quantity"
-                    ],
-                }),
-                logicSection
-            ]
+            Transfer: [basicSection, logicSection],
+            Dispense: [basicSection, logicSection],
+            Receive: [basicSection, logicSection],
         },
-        baseTransactionOptions = {
+        baseTransactionOptions = util.extend(mugs.defaultOptions, {
```

Comment:
Side note: Maybe we could change this to `util.deepExtend` so as to not confuse it with `_.extend`